mopidy_musicbox_webclient/static/js/controls.js

```diff
@@ -424,15 +424,19 @@ function triggerVolume() {
     volumeChanging = 0;
 }
 
-function toggleMute() {
-    mopidy.mixer.getMute().then(function(mute) {
-        mopidy.mixer.setMute(!mute);
+function setMute(nwmute) {
```

Comment:
It took me a while to figure out initially, but that appears to be the convention (which we are following here as well):

- ```doFoo``` is used in the HTML to trigger the actual Mopidy API calls.

- ```setFoo``` is only used to update the UI and change icons or toggle colours etc.

I agree that it would be clearer if we could change this, and maybe that is something that we can refactor across the entire codebase as part of the general bug cleanup. 

It's beyond the scope of this PR though.